src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,81 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+import static seedu.addressbook.common.Messages.*;
```

Comment:
The coding standard for CS2103 specifies to import just the specific classes you need. If this happens automatically, you can change "code style -> Java -> imports" and change the setting